public/lib/agg_response.js

```diff
@@ -10,6 +11,19 @@ module.exports = function sankeyProvider(Private, createNotifier) {
   });
 
   return function (vis, resp) {
+    // TODO: Remove Workaround
```

Comment:
No it is not, I removed it now.
The main purpose of it was just to point out that the code was adapted because of the change of data structure in Kibana.